Lesson32/Merve/src/components/Exercise8.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+
+export const Exercise8 = () => {
+    const [todos, setTodos] = useState([
+        'Finish the assignments',
+        'Study JS on freetime',
+        'Learn React',
+    ]);
+
+    const removeTodo = (index) => {
```

Comment:
It's dangerous to use index of element in the array in situations like this (deleting an element) as eventually your code will start removing incorrect elements (order of the elements in the list changes when element is removed).

Instead, it's always better to add an unique id attribute to whatever data you have and refer to it. E.g. ```
[
{"id": "b4a2c5f5-dcf8-42ff-9f09-913a68df3771",
"label": 'Finish the assignments',
}
]```

You can use UUID: https://www.uuidgenerator.net/ or if you need a quicker solution, you can use a Date.now() as it generates a timestamp in milliseconds which is quite unique.